test-exec.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
this won't work on Windows.

Reply:
I'll get this fixed. Thanks for the super fast review.

Reply:
Done.